labgrid/remote/client.py

```diff
@@ -613,18 +613,29 @@ async def release(self):
         if not place.acquired:
             raise UserError(f"place {place.name} is not acquired")
         _, user = place.acquired.split('/')
-        if user != self.getuser():
+        if user != self.getuser() and not self.args.release_from:
             if not self.args.kick:
                 raise UserError(f"place {place.name} is acquired by a different user ({place.acquired}), use --kick if you are sure")  # pylint: disable=line-too-long
             print(f"warning: kicking user ({place.acquired})")
         res = await self.call(
-            'org.labgrid.coordinator.release_place', place.name
+            'org.labgrid.coordinator.release_place', place.name, self.args.release_from
         )
         if not res:
             raise ServerError(f"failed to release place {place.name}")
 
         print(f"released place {place.name}")
 
+    async def release_from(self):
+        """Release a place, but only if acquired by a specific user"""
+        place = self.get_place()
+        res = await self.call(
+            'org.labgrid.coordinator.release_place_from', place.name, self.args.acquired,
+        )
+        if not res:
+            raise ServerError(f"failed to release place {place.name}")
+
+        print(f"{self.args.acquired} has released place {place.name}")
+
```

Comment:
Why is this still needed?

Reply:
Its not, left over and forgot to remove.